### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#193] [Esercizi - Bertini]
![Screenshot 2023-11-20 224011](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/5795f79c-f259-49d8-b126-4bafc83b6874)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/bea4b413-f98f-4c53-bf4b-a00ebfff2563)

![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/e2d8ea83-ff2d-4e4e-9eb2-c750573131f6)

**Reply by CasuFrost** [reply]
C'è un piccolo errore di calcolo nell'ultima riga, la varianza esce $\displaystyle\frac{5}{18}$

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/93aedda5-5b7c-4a62-9c85-2766a18dc6b3)
